src/test/java/software/amazon/awssdk/crt/test/SigningTest.java

```diff
@@ -472,7 +489,7 @@ private HttpRequestBodyStream createChunk2Stream() {
     private static byte[] EXPECTED_FIRST_CHUNK_SIGNATURE = "ad80c730a21e5b8d04586a2213dd63b9a0e99e0e2307b0ade35a65485a288648".getBytes(StandardCharsets.UTF_8);
     private static byte[] EXPECTED_SECOND_CHUNK_SIGNATURE = "0055627c9e194cb4542bae2aa5492e3c1575bbb81b612b7d234b86a503ef5497".getBytes(StandardCharsets.UTF_8);
     private static byte[] EXPECTED_FINAL_CHUNK_SIGNATURE = "b6c6ea8a5354eaf15b3cb7646744f4275b71ea724fed81ceb9323e279d449df9".getBytes(StandardCharsets.UTF_8);
-    private static byte[] EXPECTED_TRAILING_HEADERS_SIGNATURE = "17b7a1e28961dba05b733dbdbe8ea230d8bed5ea507bc8af6f76349207c05315".getBytes(StandardCharsets.UTF_8);
+    private static byte[] EXPECTED_TRAILING_HEADERS_SIGNATURE = "0248ea7d40ea8edfeb417c5aab802f40ccf25c1f301f4f24d9032dea49bea897".getBytes(StandardCharsets.UTF_8);
```

Comment:
I don't really understand. Is that our previous expected signature wrong or what?

Reply:
change in trailing header implementation, changed the value of the constant